core/tauri/src/command/authority.rs

```diff
@@ -112,9 +112,27 @@ impl<T: Debug + DeserializeOwned + Send + Sync + 'static> ScopeValue<T> {
   }
 }
 
+#[derive(Debug)]
+enum OwnedOrRef<'a, T: Debug> {
```

Comment:
Can't we use `Cow` ?

Reply:
Cow requires `T: Clone` and we don't really need that :(

Reply:
I see, it is fine then